app/models/form/sales/pages/owning_organisation.rb

```diff
@@ -20,11 +20,13 @@ def routed_to?(log, current_user)
     if current_user.organisation.holds_own_stock?
       return true if current_user.organisation.absorbed_organisations.any?(&:holds_own_stock?)
       return true if stock_owners.count >= 1
+      return false if log.owning_organisation == current_user.organisation
 
       log.update!(owning_organisation: current_user.organisation)
```

Comment:
This feels like an irrelevant update if it's already set to that (was failing some tests)